xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/notification/groupMessage.vm

```diff
@@ -0,0 +1,26 @@
+## ---------------------------------------------------------------------------
+## See the NOTICE file distributed with this work for additional
+## information regarding copyright ownership.
+##
+## This is free software; you can redistribute it and/or modify it
+## under the terms of the GNU Lesser General Public License as
+## published by the Free Software Foundation; either version 2.1 of
+## the License, or (at your option) any later version.
+##
+## This software is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## Lesser General Public License for more details.
+##
+## You should have received a copy of the GNU Lesser General Public
+## License along with this software; if not, write to the Free
+## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+## ---------------------------------------------------------------------------
+## TODO: find a way to move it to the "message stream" module
+#template('notification/macros.vm')
+#set ($translationParameters = [
+    "#displayNotificationEventUser($event.events[0].user, false)",
+    "#displayNotificationEventUser($event.events[0].stream, false)"
+])
+#displayMessageNotification($event, 'messagestream.notification.groupMessage.description', 'group', $translationParameters)
```

Comment:
Don't we need a notification RSS template too?

Reply:
Indeed I missed that. I've tested the message stream RSS and it is not well handled for all the types of messages, with displayed contents looking like this:

Title: \*\*\* No title \*\*\*
Content:
```MessageStream: [publicMessage] cd94f7c8-af2b-4c78-92e9-34ca168fb125 .
edited by Administrator.

2020/10/08 09:28 
Changes: ${newdoc.version}```

Might be better to create another issue for this one.

Reply:
I already created an issue for the title problem in RSS notifications templates for RecordableEvent (https://jira.xwiki.org/browse/XWIKI-17851) but I guess we need a specific handle of it for MessageStream events.

Reply:
PR for the improved display of the message stream rss entries https://github.com/xwiki/xwiki-platform/pull/1419